### PR TITLE
Feature/base architecture

### PR DESCRIPTION
**Pull request by SashaMatsun**
cycle set_position added

### CODE REVIEW

**Comment by i-khokhlov** [comment]
Delete __pycache__

File: controller/imu_sensor_interface.py [hunk 1]
```diff
@@ -1,15 +1,14 @@
...
-import rclpy
 
 
 class IMUSensorInterface(SensorInterface):
     def __init__(self, name) -> None:
         super().__init__(name)
         self._frame_id = self._name
-        self.__publisher = self.create_publisher(Imu, 'imu_sensor', 1)
-        self.__current_imu = None
+        #self.__publisher = self.create_publisher(Imu, 'imu_sensor', 1)
```
Comment (vorkorro14) [inline controller/imu_sensor_interface.py:10]: No unused code as comments

File: controller/hw_interface.py [hunk 2]
```diff
@@ -16,8 +12,8 @@ def start(self):
     def stop(self):
         raise NotImplementedError
 
-    #def get_name(self):
-    #    return self._name
+    def get_name(self):
+        return self._name
 
     def get_status(self):
         return self._status
```
Comment (vorkorro14) [inline controller/hw_interface.py:19]: New line at the end

File: controller/position_controller_interface.py [hunk 3]
```diff
@@ -1,19 +1,48 @@
...
 
 class PositionControllerInterface(ControllerInterface):
-    def __init__(self):
-        pass
+    def __init__(self, name):
+        super().__init__(name)
+        self._pos_queue = queue.Queue()
 
-    def step(self):
+        self._servo_id_list = ['head_yaw',
```
Comment (vorkorro14) [inline controller/position_controller_interface.py:9]: Better to read from file

**Review by vorkorro14 — APPROVED** [review]
Add .gitignore file to avoid pushing of __pycache__ folders and other unused stuff

**Pull request merged**